test/controllers/git-tab-controller.test.js

```diff
@@ -105,30 +107,26 @@ describe('GitTabController', function() {
   });
 
   it('displays the staged changes since the parent commit when amending', async function() {
-    const didChangeAmending = sinon.spy();
     const workdirPath = await cloneRepository('multiple-commits');
     const repository = await buildRepository(workdirPath);
     const ensureGitTab = () => Promise.resolve(false);
+    // TODO: check that git tab opens in root controller when mrege conflict
```

Comment:
Ooh I like that this is a thing we can do, now.

Reply:
Also `mrege` -> `merge`.